service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/base/StandardDirectionOrderContent.java

```diff
@@ -1,33 +1,24 @@
 package uk.gov.hmcts.reform.fpl.service.email.content.base;
 
-import com.google.common.collect.ImmutableMap;
 import uk.gov.hmcts.reform.fpl.model.CaseData;
 
 import java.time.format.FormatStyle;
 
 import static java.util.Objects.isNull;
-import static org.apache.commons.lang3.StringUtils.capitalize;
 import static uk.gov.hmcts.reform.fpl.utils.DateFormatterHelper.formatLocalDateToString;
-import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
 
 public abstract class StandardDirectionOrderContent extends AbstractEmailContentProvider {
 
-    protected ImmutableMap.Builder<String, Object> getSDOPersonalisationBuilder(CaseData caseData) {
-        return ImmutableMap.<String, Object>builder()
-            .put("familyManCaseNumber",
-                isNull(caseData.getFamilyManCaseNumber()) ? "" : caseData.getFamilyManCaseNumber() + ",")
-            .put("leadRespondentsName", capitalize(caseData.getRespondents1()
-                .get(0)
-                .getValue()
-                .getParty()
-                .getLastName()))
-            .put("hearingDate", getHearingBooking(caseData))
-            .put("reference", String.valueOf(caseData.getId()))
-            .put("callout", buildCallout(caseData))
-            .put("caseUrl", getCaseUrl(caseData.getId(), "OrdersTab"));
+    protected String getFamilyManCaseNumber(CaseData caseData) {
+        return isNull(caseData.getFamilyManCaseNumber()) ? "" : caseData.getFamilyManCaseNumber() + ",";
     }
 
-    private String getHearingBooking(CaseData data) {
+    protected String getLeadRespondentsName(CaseData caseData) {
```

Comment:
I feel like this method can be inlined and the callers instead just use the method that this one is using